.github/workflows/cache-assets.yaml

```
@@ -0,0 +1,46 @@
+on:
+  push:
+    branches:
+      - main
+  workflow_dispatch:
+
+jobs:
+  download_artifact:
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v4
+      with:
+        ref: assets
+        path: br-assets
+
+    - name: Download artifacts from falco
+      uses: dawidd6/action-download-artifact@v2.28.0
+      with:
+        github_token: ${{ github.token }}
+        workflow: 26039754
+        repo: falcosecurity/falco
+        workflow_conclusion: success
+        name: falco-\d+\.\d+\.\d+-\d+\+[a-f0-9]+-wasm.tar.gz
+        name_is_regexp: true
```

```suggestion
         github_token: ${{ secrets.GITHUB_TOKEN }}
          workflow: "ci.yml"
          repo: "falcosecurity/falco"
          name: falco-\d+\.\d+\.\d+-\d+\+[a-f0-9]+-wasm.tar.gz
          name_is_regexp: true
```
Let's remove the hard-coded `run id`. Having a specific run_id will fail in the long run since artifacts have a retention period of 30 days.

---

Addressed here: https://github.com/falcosecurity/falco-playground/pull/17

---

that's the workflow ID, it's referring to the same "CI Build" workflow, I'm taking the latest successful run's artefacts

---

oh ok, can you please change it to `ci.yml` to keep it consistent 😄